daniel-v2/README.md

```diff
@@ -27,6 +28,12 @@
   - ✅✨ [Sitemap](https://wpdeveloperstaging.wordpress.com/docs/platform-features/sitemaps/)
   - ✅✨ [Jetpack Scan](https://wpdeveloperstaging.wordpress.com/docs/platform-features/jetpack-scan/)
   - ✅✨ [Account Security](https://wpdeveloperstaging.wordpress.com/docs/platform-features/account-security/)
+  - 🚧 [Automatic Data Center Failover](https://github.com/Automattic/dotcom-content/issues/218)
+  - 🚧 [Distributed Denial of Service (DDOS) and Web Application Firewall (WAF) protection](https://github.com/Automattic/dotcom-content/issues/219)
```

Comment:
This item seems quite long compared to the others. Does it make sense to split them up since they're different items?